18 Runners/.circleci/config.yml

```diff
@@ -0,0 +1,11 @@
+version: 2.1
+
+jobs:
+    # Add your config here
```

Comment:
If someone copy/pastes this config file and adds the proper values to run on Runner, it will result in a build error since it's missing the job name. Can we adjust this example config to the following instead?

```
version: 2.1

jobs:
  runner:
    # Add your config here
    steps:
      - run: echo "Hi I'm on Runners!"

workflows:
  testing:
    jobs:
      - runner
```